### DISCUSSION TITLE
windhalk não funciona

### DISCUSSION THREAD

**Discussion by h4ll3y1202** [ramensoftware/windhawk#82] [Q&A]
Meu windhalk não funciona.
ele funcionava até 5 dias atrás, e não funciona mais, já tentei de tudo mas nada funciona, já instalei e desinstalei várias vezes mas não funciona, ele até baixa os mods, mas nenhum mod funciona]
sem mais,
halley

**Comment by m417z** [comment]
Was resolved in Discord. 

![Screenshot](https://github.com/ramensoftware/windhawk/assets/4129781/cf3a3e4d-88b5-4824-829e-3ff2c7515a30)
